tests/Fixtures/Integration/misc/phpdoc_to_comment,phpdoc_var_without_name.test

```diff
@@ -8,19 +8,35 @@ Integration of fixers: phpdoc_to_comment,phpdoc_var_without_name.
 --EXPECT--
 <?php
 
-$first = true;// needed because by default first docblock is never fixed.
+/**
```

Comment:
What is the point of this test? `phpdoc_to_comment` is not doing anything here (neither it was before changes).

Maybe... maybe we can have a docblock with a var name to remove, and `phpdoc_to_comment` would change it to comment before `phpdoc_var_without_name` would remove the name - but personally I'd remove this test.

Reply:
This is exactly why I initially deleted this test, however @keradus was totally against it, as can be seen [in the discussion above](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3911#discussion_r202309642).